### ISSUE TITLE
Cannot import Concordium Node SDK: "Cannot find module '../grpc/concordium_p2p_rpc_grpc_pb'"

### ISSUE THREAD

**Issue by iicc1** [Concordium/concordium-node-sdk-js#68]
**Bug Description**
I'm not able to use the Concordium Node SDK since Node throws an error when loading the package.

**Steps to Reproduce**
```
# Install the packages
npm i @grpc/grpc-js @concordium/node-sdk

# Create a Nodejs script
touch concordium.js

# Edit the script including the imports and save it
nano concordium.js

import { credentials, Metadata } from '@grpc/grpc-js'
import { ConcordiumNodeClient } from '@concordium/node-sdk'

# Run the script
node concordium.js
```

**Expected Result**
The script should execute without errors

**Actual Result**
Nodejs throws an error:
```
node:internal/modules/cjs/loader:927
  const err = new Error(message);
              ^

Error: Cannot find module '../grpc/concordium_p2p_rpc_grpc_pb'
Require stack:
- /home/stakely/tmp/node_modules/@concordium/node-sdk/lib/client.js
- /home/stakely/tmp/node_modules/@concordium/node-sdk/lib/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:927:15)
    at Function.Module._load (node:internal/modules/cjs/loader:772:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/home/stakely/tmp/node_modules/@concordium/node-sdk/lib/client.js:4:38)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1124:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:816:12)
    at Module.require (node:internal/modules/cjs/loader:999:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/stakely/tmp/node_modules/@concordium/node-sdk/lib/client.js',
    '/home/stakely/tmp/node_modules/@concordium/node-sdk/lib/index.js'
  ]
}
```

**Versions**
- Software Version: 
Nodejs 16
@concordium/node-sdk 2.0.0
@grpc/grpc-js: 1.6.7

- OS
Windows and Ubuntu

**Comment by parv0888** [comment]
Also that this repo does not have a listed release 2.0.0 while the node sdk has a release 2.0.0.

**Comment by shjortConcordium** [comment]
Thanks for creating this issue. We have released a 2.0.2 version that fixes this issue, so please update to that and try again.

> Also that this repo does not have a listed release 2.0.0 while the node sdk has a release 2.0.0.
We won't be listing releases on here after making it a mono-repo with multiple packages. (We'll still use tags for the different versions)
